Description: Using six.u() for unicode strings in tests
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2014-11-25

--- python-pymysql-0.6.2.orig/pymysql/tests/test_basic.py
+++ python-pymysql-0.6.2/pymysql/tests/test_basic.py
@@ -1,3 +1,5 @@
+import six
+
 import pymysql.cursors
 
 from pymysql.tests import base
@@ -18,7 +20,7 @@ class TestConversion(base.PyMySQLTestCas
         c.execute("create table test_datatypes (b bit, i int, l bigint, f real, s varchar(32), u varchar(32), bb blob, d date, dt datetime, ts timestamp, td time, t time, st datetime)")
         try:
             # insert values
-            v = (True, -3, 123456789012, 5.7, "hello'\" world", u"Espa\xc3\xb1ol", "binary\x00data".encode(conn.charset), datetime.date(1988,2,2), datetime.datetime.now(), datetime.timedelta(5,6), datetime.time(16,32), time.localtime())
+            v = (True, -3, 123456789012, 5.7, "hello'\" world", six.u("Espa\xc3\xb1ol"), "binary\x00data".encode(conn.charset), datetime.date(1988,2,2), datetime.datetime.now(), datetime.timedelta(5,6), datetime.time(16,32), time.localtime())
             c.execute("insert into test_datatypes (b,i,l,f,s,u,bb,d,dt,td,t,st) values (%s,%s,%s,%s,%s,%s,%s,%s,%s,%s,%s,%s)", v)
             c.execute("select b,i,l,f,s,u,bb,d,dt,td,t,st from test_datatypes")
             r = c.fetchone()
@@ -104,9 +106,9 @@ class TestConversion(base.PyMySQLTestCas
         conn = self.connections[0]
         c = conn.cursor()
         c.execute("select null,''")
-        self.assertEqual((None,u''), c.fetchone())
+        self.assertEqual((None,six.u('')), c.fetchone())
         c.execute("select '',null")
-        self.assertEqual((u'',None), c.fetchone())
+        self.assertEqual((six.u(''),None), c.fetchone())
 
     def test_timedelta(self):
         """ test timedelta conversion """
--- python-pymysql-0.6.2.orig/pymysql/tests/test_issues.py
+++ python-pymysql-0.6.2/pymysql/tests/test_issues.py
@@ -1,3 +1,5 @@
+import six
+
 import pymysql
 from pymysql.tests import base
 try:
@@ -110,9 +112,9 @@ KEY (`station`,`dh`,`echeance`)) ENGINE=
         c.execute("drop table if exists issue15")
         c.execute("create table issue15 (t varchar(32))")
         try:
-            c.execute("insert into issue15 (t) values (%s)", (u'\xe4\xf6\xfc',))
+            c.execute("insert into issue15 (t) values (%s)", (six.u('\xe4\xf6\xfc'),))
             c.execute("select t from issue15")
-            self.assertEqual(u'\xe4\xf6\xfc', c.fetchone()[0])
+            self.assertEqual(six.u('\xe4\xf6\xfc'), c.fetchone()[0])
         finally:
             c.execute("drop table issue15")
 
